lib/kitchen/transport/winrm_file_transfer/remote_zip_file.rb

```diff
@@ -92,6 +93,7 @@ def extract_zip_command
             $shellApplication = new-object -com shell.application
 
             $zipPackage = $shellApplication.NameSpace('#{remote_path}')
+            Remove-Item -Recurse -Force $dest | Out-Null
```

Comment:
I think we should have either a `-ErrorAction SilentlyContinue` or wrap in a `Test-Path` since I believe this will result in an error if the directory does not exist. Pretty sure its a "non terminating" error but it may cause unnecessary confusion to see the error text.